engine.py

```diff
@@ -176,6 +188,27 @@ def _get_dialog_parent(self):
 
         return window
 
+    def _hide_menu_in_scripts(self):
+        """
+        Remove the entry in the VRED Scripts window
+        """
+        main_window = self._get_dialog_parent()
+        menu_names = main_window.menuBar().actions()
+
+        # Make it invisible for both Pro & Design
+        for menu_name in menu_names:
```

Comment:
Assuming VRED only has one menu action with name `"Scripts"` we can stop iterating once we've found the Scripts menu action. We could find the first occurrence like so:

```
scripts_action = next((menu_action for menu_action in menu_actions if menu_action.text() == "Scripts"), None)
if scripts_action:
    # find Shotgun menu and hide
```

We can use `next` to similarly find the Shotgun menu as well, assuming there is only one Shotgun menu